NGitLab.Mock/Clients/CommitClient.cs

```diff
@@ -38,5 +42,20 @@ public JobStatus GetJobStatus(string branchName)
         {
             throw new NotImplementedException();
         }
+
```

Comment:
Hi @meziantou @louis-z and @Toa741, I would like you to check this file especially please.

Reply:
Looks ok :)

Reply:
Thanks!